tests/scripts/kubeadm.sh

```diff
@@ -59,8 +60,13 @@ EOF
 #install k8s node
 install_node(){
     echo "inside install node function"
-    echo "kubeadm join ${1} ${2} ${3} --skip-preflight-checks"
-    sudo kubeadm join ${1} ${2} ${3} --skip-preflight-checks || true
+    if [[ $KUBE_VERSION == v1.5* ]] || [[ $KUBE_VERSION == v1.6* ]] || [[ $KUBE_VERSION == v1.7* ]]; then
```

Comment:
i thought we weren't worrying about v1.5?

Reply:
this is fixed now.